tests/browser-automated-tests-playwright/tests/clear_connect_account_id.js

```diff
@@ -0,0 +1,12 @@
+const { test, expect } = require('@playwright/test');
```

Comment:
Please `git mv clear_connect_account_id.js delete_stripe_connect_account.js`
Why? 
Because that's what's happening - reduces ambiguity